### PR TITLE
Java cc31

### PR DESCRIPTION
**Pull request by wafman**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401Challenges/src/main/java/code401Challenges/RepeatedWord/RepeatedWord.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        if(words.length < 1){
+            throw new NullPointerException("empty string");
+        }
+
+        HashMap<String, Integer> hashMap = new HashMap<>();
+
+        for(String word : words){
+            //regex from stack overflow, see readme for link.
+            temp = word.toLowerCase().replaceAll("[^a-zA-Z]+$", "");
```
Comment (evanslaton) [inline code401Challenges/src/main/java/code401Challenges/RepeatedWord/RepeatedWord.java]: Because you are using `.toLowerCase()` the `A-Z` in your regex isn't necessary (it doesn't hurt though)
